prisma/seed/conversations.ts

```
@@ -0,0 +1,151 @@
+
+  // ── storeA: 커스텀 인사말 + FAQ 칩 ──
+  await prisma.store.update({
+    where: { id: storeA.id },
+    data: {
+      greeting_message:
+        '안녕하세요! {nickname} 고객님.\n{storeName} 입니다 😄\n무엇을 도와드릴까요?',
+    },
+  });
+
+  const faqRows = [
+    { title: '날짜 변경', answer_html: '<p>픽업 1일 전까지 채팅으로 요청해 주시면 일정 확인 후 변경해 드려요.</p>' },
```

ESLint

[error] Replace `·title:·'날짜·변경',·answer_html:·'<p>픽업·1일·전까지·채팅으로·요청해·주시면·일정·확인·후·변경해·드려요.</p>'` with `⏎······title:·'날짜·변경',⏎······answer_html:⏎········'<p>픽업·1일·전까지·채팅으로·요청해·주시면·일정·확인·후·변경해·드려요.</p>',⏎···`

(prettier/prettier)